edx_sysadmin/__init__.py

```diff
@@ -3,6 +3,6 @@
 An Open edX plugin to enable SysAdmin panel.
 """
 
-__version__ = "0.2.2"
```

Comment:
I ended up manually releasing this version, so we don't need to revert the version. Because we are adding this commit we can just use Doof to bump to 0.2.3

Reply:
Oh. Reverted these changes in https://github.com/mitodl/edx-sysadmin/pull/83/commits/f87add00f75fcc577fe4fcd85307b8cfc0da741f.